pyfar/signal.py

```diff
@@ -343,10 +343,18 @@ def __repr__(self):
         """String representation of signal class.
         """
         repr_string = (
-            "Audio Signal\n"
-            "--------------------\n"
-            "{} channels with {} samples @ {} Hz sampling rate".format(
-                self.cshape, self.n_samples, self._sampling_rate))
+            f"{self.domain} domain audio Signal\n"
+            "--------------------------------\n"
+            f"{self.cshape} channels with {self.n_samples} samples @ "
+            f"{self._sampling_rate} Hz sampling rate\n"
+            f"Sinal type: {self.signal_type}\n"
+            f"FFT normalisation: {self.fft_norm}\n"
+            f"Comment: ")
```

Comment:
I think I would rather keep the `__repr__`  compact.
The comment might increase terminal spam too much, probably resulting in users not using it.

Reply:
I think I would also be in favor of using as few lines as possible.